substrate/frame/revive/fixtures/contracts/unstable_interface.rs

```diff
@@ -0,0 +1,44 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#![no_std]
+#![no_main]
+
+extern crate common;
+
+#[polkavm_derive::polkavm_import]
+extern "C" {
+	pub fn set_code_hash();
+}
```

Comment:
shouldn't we make a dedicated unstable api just for test so that this does not fail when this becomes stable

Reply:
Yeah that is a good idea. But we will cross the bridge once we get there.